### PR TITLE
On 1857 application of permissions

### PR DESCRIPTION
**Pull request by davidgouge**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Application/Handlers/UserManagement/RequestForPrivilegeHandler.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        {
+            _mediator = mediator;
+            _contactQueryRepository = contactQueryRepository;
+            _organisationQueryRepository = organisationQueryRepository;
+            _config = config;
+            _contactRepository = contactRepository;
+        }
+        
+        public async Task Handle(RequestForPrivilegeRequest message, CancellationToken cancellationToken)
+        {
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.Application/Handlers/UserManagement/RequestForPrivilegeHandler.cs]: Do you need to worry if anything returns null (i.e. it cannot find the contact or the privilege for example) ?

File: src/SFA.DAS.AssessorService.Database/Tables/Privileges.sql [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ﻿CREATE TABLE [dbo].[Privileges](
 	[Id] [uniqueidentifier] NOT NULL,
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.Database/Tables/Privileges.sql]: Could we make Id have a Default of NEWID() ?

File: src/SFA.DAS.AssessorService.Application/Handlers/UserManagement/RequestForPrivilegeHandler.cs (hunk 1 above)
Comment (davidgouge) [reply src/SFA.DAS.AssessorService.Application/Handlers/UserManagement/RequestForPrivilegeHandler.cs]: Not in this case as we are already sure the contact and privilege exist as the Contact is the one trying to access the page (so must exist) and the privilege is what's stopped the contact accessing the page (so must exist). 😁

File: src/SFA.DAS.AssessorService.Database/Tables/Privileges.sql (hunk 2 above)
Comment (davidgouge) [reply src/SFA.DAS.AssessorService.Database/Tables/Privileges.sql]: Possibly in the future, but for now these Privileges aren't created anywhere in code and are just 'hard coded' into the db. 👍

**Pull request merged**